.github/dockerfiles/ov_build/ubuntu_22_04_x64_dpcpp/Dockerfile

```diff
@@ -0,0 +1,94 @@
+FROM openvinogithubactions.azurecr.io/dockerhub/ubuntu:22.04
+
+USER root
+
+# APT configuration
+RUN echo 'Acquire::Retries "10";' > /etc/apt/apt.conf && \
+    echo 'APT::Get::Assume-Yes "true";' >> /etc/apt/apt.conf && \
+    echo 'APT::Get::Fix-Broken "true";' >> /etc/apt/apt.conf && \
+    echo 'APT::Get::no-install-recommends "true";' >> /etc/apt/apt.conf
+
+ENV DEBIAN_FRONTEND="noninteractive" \
+    TZ="Europe/London"
+
+RUN apt-get update && \
+    apt-get install software-properties-common && \
+    add-apt-repository --yes --no-update ppa:git-core/ppa && \
+    add-apt-repository --yes --no-update ppa:deadsnakes/ppa && \
+    apt-get update && \
+    apt-get install \
+        wget \
+        curl \
+        git \
+        ca-certificates \
+        gpg-agent \
+        tzdata \
+        libtbb2 \
+        # Pythons \
+        python3.11-dev \
+        python3.11-venv \
+        python3.11-distutils \
```

Comment:
you need to install Java SDK as build job also builds Contrib
https://github.com/openvinotoolkit/openvino/blob/54f58b86be2f7b15159abffd5544477276342ac9/.github/dockerfiles/ov_build/ubuntu_22_04_x64/Dockerfile#L29-L30